cfgov/cal/admin.py

```diff
@@ -28,21 +32,23 @@ def start_date(self, obj):
         return obj.dtstart
     start_date.short_description = 'Start Date'
 
+
 class CFPBPDFFileAdmin(admin.ModelAdmin):
     ordering = ['-id']
 
+
 class CFPBCalendarImportICSFileAdmin(admin.ModelAdmin):
 
     def changelist_view(self, request, extra_context=None):
         context = {}
         context.update(extra_context or {})
-        context.update({ 'icsfile': None,})
-        context.update({ 'calendar_record': None,})
-        context.update({ 'processed_events': None,})
+        context.update({'icsfile': None})
+        context.update({'calendar_record': None})
+        context.update({'processed_events': None})
 
         leadership = CFPBCalendar.objects.all().order_by("title")
 
-        context.update({ 'leadership': leadership,})
+        context.update({'leadership': leadership, })
```

Comment:
Yeah, I agree these should be a single context.update. I purposefully didn't do that with this PR though.